pkg/storage/utils/decomposedfs/tree/tree.go

```diff
@@ -148,15 +148,15 @@ func (t *Tree) Setup(owner string) error {
 				// we can reuse the node id as the space id
 				err = os.Symlink("../../nodes/"+nodes[i].Name(), filepath.Join(t.root, "spaces/personal", nodes[i].Name()))
 				if err != nil {
-					fmt.Printf("could not create symlink for personal space %s, %s\n", nodes[i].Name(), err)
+					//fmt.Printf("could not create symlink for personal space %s, %s\n", nodes[i].Name(), err)
```

Comment:
urgh ... no log available here, right?

Reply:
you could use
```
	import "github.com/cs3org/reva/pkg/logger"

	logger.New().Error().Err(err).Msg("could not setup tree")
```

but it will have the default log level etc ...